modules/src/__init__.py

```diff
@@ -4,6 +4,7 @@
     'book',
     'bye',
     'coin',
+    'cpcontest',
```

Comment:
"contest" would be too generalized, other names are too big such as "programingcontest" or something like that. So, I abbreviated "competitive programming" to "cp" here. Do you have any suggestion?